handler/src/main/java/io/netty/handler/ssl/SslHandler.java

```diff
@@ -851,8 +851,8 @@ private boolean ignoreException(Throwable t) {
                             && "com.sun.nio.sctp.SctpChannel".equals(clazz.getSuperclass().getName())) {
                         return true;
                     }
-                } catch (ClassNotFoundException e) {
-                    // This should not happen just ignore
+                } catch (Throwable cause) {
+                    logger.debug("Unexpected exception", cause);
```

Comment:
@Scottmitch can you use some better logging message ?

Maybe something like `Unexpected exception while loading class {}`, class, cause)`